packages/shared/src/lib/core/ledger/classes/ledger.class.ts

```diff
@@ -4,14 +4,15 @@ import { IPlatformEventMap } from '@core/app/interfaces'
 import { localize } from '@core/i18n'
 import { IEvmAddress, IEvmTransactionSignature } from '@core/layer-2/interfaces'
 import { EvmTransactionData } from '@core/layer-2/types'
-import { prepareEvmTransaction } from '@core/layer-2/utils'
+import { getAmountFromEvmTransactionValue, prepareEvmTransaction } from '@core/layer-2/utils'
 import { MILLISECONDS_PER_SECOND, sleep } from '@core/utils'
 
 import { DEFAULT_LEDGER_API_REQUEST_OPTIONS } from '../constants'
 import { LedgerApiMethod } from '../enums'
 import { ILedgerApiBridge } from '../interfaces'
 import { LedgerApiRequestResponse } from '../types'
 import type { Bip44 } from '@iota/wallet/types'
+import { closePopup, openPopup, PopupId } from 'desktop/lib/auxiliary/popup'
```

Comment:
shouldnt it be 
```suggestion
import { closePopup, openPopup, PopupId } from '@desktop/lib/auxiliary/popup'
```
or
```suggestion
import { closePopup, openPopup, PopupId } from '../../../../lib/auxiliary/popup'
```